hydroflow_datalog_core/src/grammar.rs

```diff
@@ -30,12 +32,17 @@ pub mod datalog {
     pub struct RustSnippet {
         #[rust_sitter::leaf(text = "`")]
         _start: (),
-        #[rust_sitter::leaf(pattern = r#"[^`]*"#, transform = |s| s.to_string())]
-        pub code: String,
+        pub code: Spanned<RustSnippetInner>,
```

Comment:
released Rust Sitter 0.3.1 which should make it possible to go back to just `code: Spanned<String>`